core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -163,6 +163,8 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
                     style="margin: 0 1px;">
               </span>
               <span translate="<[activeSubsection]>"></span>
+              <i ng-if="activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.COLLECTIONS" class="oppia-tooltip material-icons">info</i>
+              <span class="oppia-tooltip-text">Collections are multiple related explorations that are intended to be completed in a sequence.</span>
```

Comment:
This text should use the translate system so that it can be converted into other languages.